src/components/CredderRating/FeedCredderRating.js

```diff
@@ -58,10 +58,10 @@ const FeedCredderRating = ({
 
   const renderCredderRatingScore = () => {
     if (!score || score < 0) return 'n/a';
-    // return ${score}${<Text>{`%`}</Text>}
+    const parsedScore = Math.round(parseInt(score, 10));
     return (
       <Text>
-        {`${score}`}
+        {`${parsedScore}`}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for ensuring that the score is parsed as an integer and rounded before being displayed. However, it's important to handle potential exceptions that could occur during parsing. If `score` is not a number, `parseInt(score, 10)` will return `NaN`, and `Math.round(NaN)` is also `NaN`. Displaying `NaN` to users would not be ideal. Therefore, you should add error handling to ensure that `parsedScore` is a valid number before displaying it.
```diff
-    const parsedScore = Math.round(parseInt(score, 10));
+    let parsedScore = parseInt(score, 10);
+    if (isNaN(parsedScore)) {
+        return 'n/a';
+    }
+    parsedScore = Math.round(parsedScore);
```
This way, if the score cannot be parsed into a number, the function will return 'n/a', similar to when the score is not available or less than 0.


<!-- This is an auto-generated comment by OSS CodeRabbit -->